dbt/adapters/redshift/impl.py

```diff
@@ -73,7 +73,7 @@ def _behavior_flags(self) -> List[BehaviorFlag]:
         return [
             {
                 "name": "restrict_direct_pg_catalog_access",
-                "default": False,
+                "default": True,
```

Comment:
This allows us to run all tests as if the flag were enabled without adjusting the tests. This should be reverted prior to merging.